src/cljs/dato/lib/core.cljs

```diff
@@ -85,35 +81,32 @@
         ss-api             (atom {})
         ;; ???: Do the perf implications of anonymous functions mean these
         ;; shouldn't be in a tight loop?
-        fn-desc->rpc       (fn fn-desc->rpc [remote-name desc]
-                             (let [f (fn [& all-args]
-                                       ;; TODO: Check if last argument is a channel, an
-                                       (let [cb-or-ch (last all-args)
-                                             cb       (cond
-                                                        (fn? cb-or-ch)   cb-or-ch
-                                                        (chan? cb-or-ch) #(put! cb-or-ch %)
-                                                        :else            false)
-                                             args     (if cb
-                                                        (drop-last 1 all-args)
-                                                        all-args)
-                                             uuid     (d/squuid)]
-                                         (when (fn? cb)
-                                           (swap! pending-rpc assoc uuid cb-or-ch))
-                                         (dato-send! remote-name {:args   args
-                                                                  :rpc/id uuid})
-                                         (if (chan? cb-or-ch)
-                                           cb-or-ch
-                                           uuid)))]
-                               f))]
+        fn-desc->rpc       (fn [remote-name desc]
```

Comment:
The reason this was named before is for stack traces - it's nice to have a named function